topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md

```diff
@@ -661,26 +680,31 @@ the datasets number 62 and 63.
 > >
 > > > ### {% icon solution %} Solution
 > > >
-> > > Knowing that there are 3 pools in the QC class, we can determine there are 4738 ions that have been found in all pools.
+> > > Knowing that there are 3 pools in the QC class, we can determine there are 4746 ions that have been found in all pools.
 > > >
 > > {: .solution}
 > >
 > {: .question}
 >
 {: .tip}
 
+When looking back at the plots from plotChromPeakDensity.pdf, we can notice that in some cases there seems to be a small drift of retention time for some samples. 
+This phenomenon is well known with LC-MS techniques. To be able to attribute correct groups for peaks, it may be needed to perform some retention time 
+correction accross samples. Thus, the idea is (when needed) to apply a retention time strategy on the output of your grouping step, then to perform 
+a second grouping step on the corrected data.
 
 # Optional XCMS step: *retention time correction*
 
 Sometimes with LC-MS techniques, a deviation in retention time occurs from a sample to another. In particular, this is likely to be observed when you
 inject large sequences of samples.
 
-This optional step aims to correct retention time drift for each peak among samples. The correction is based on what is called *well behaved peaks*,
-that are peaks found in all samples or at least in most of the samples.
-
-[Check slides from 19 to 21](../../tutorials/lcms-preprocessing/slides.html),
-you will find additional material to help you understand the retention time correction algorithm.
+This optional step aims at correcting retention time drift for each peak among samples. The XCMS package provides two algorithms to do so. 
+[Check the next 4 slides](../../tutorials/lcms-preprocessing/slides.html#adjustrtime),
```

Comment:
I found 5 slides on RT correction

Reply:
The fifth one concerns before/after comparison, it is referenced separately later.